src/proc_image_processing/cpu/filters/adaptive_threshold.h

```diff
@@ -6,7 +6,7 @@
 #ifndef PROVIDER_VISION_FILTERS_ADAPTIVE_THRESHOLD_H_
 #define PROVIDER_VISION_FILTERS_ADAPTIVE_THRESHOLD_H_
```

Comment:
PROVIDER_VISION_* should be changed for PROC_IMAGE_PROCESSING_* in all files of the module

Reply:
Je l'ai fixed dans la upstream branch. sauf que rendu là, ce serait probablement plus simple de merge les deux ensemble. Y'a plein de false positive dans cette branche juste parce que j'ai déplacer des fichiers